example-core/example-api/src/main/java/org/apache/shardingsphere/example/core/api/DataSourceUtil.java

```diff
@@ -29,7 +29,7 @@
     
     private static final String USER_NAME = "root";
     
-    private static final String PASSWORD = "";
+    private static final String PASSWORD = "liya76133951";// changed by liya
```

Comment:
password should not be modified.

Reply:
Sorry for that.I used it for test many days ago.I will check every file before I pull from now on.